addons/hr_evaluation/models/hr_evaluation.py

```diff
@@ -0,0 +1,410 @@
+# -*- coding: utf-8 -*-
+import calendar
+import datetime
+import uuid
+import urlparse
+from dateutil import parser
+from dateutil.relativedelta import relativedelta
+from openerp import api, fields, tools, models, _
+from openerp.exceptions import UserError
+
+
+class HrEvaluation(models.Model):
+    _name = "hr_evaluation.evaluation"
+    _inherit = ['mail.thread']
+    _description = "Employee Appraisal"
+    _order = 'date_close, interview_deadline'
+
+    _track = {
+        'state': {
+            'hr_evaluation.mt_appraisal_new': lambda self, cr, uid, obj, ctx = None: obj.state == 'new',
+        }
+    }
+    EVALUATION_STATE = [
+        ('new', 'To Start'),
+        ('pending', 'Appraisal Sent'),
+        ('done', 'Done')
+    ]
+
+    @api.model
+    def _default_set_template(self):
```

Comment:
I have removed this method and used lambda function to set default value in the field definition.